### ISSUE TITLE
Key named with a single letter 'n' fails the parser

### ISSUE THREAD

**Issue by rafi** [bronze1man/yaml2json#13]
I'm getting an error if there's a dictionary and a key named `n`.

For example:

```sh
# ERROR:
$ echo 'n: bar' | yaml2json
type mismatch: expect map key string or int; got: bool

# GOOD:
$ echo 'a: bar' | yaml2json
{"a":"bar"}
```

**Comment by bronze1man** [comment]
This bug have been fixed in v1.3 . 
Please report bugs if you find them.

**Comment by rafi** [comment]
Thanks, works great now with complex YAMLs.
